pkg/miniogw/gateway-storj.go

```diff
@@ -247,6 +247,41 @@ func (s *storjObjects) MakeBucketWithLocation(ctx context.Context,
 	return err
 }
 
+func (s *storjObjects) CopyObject(ctx context.Context, srcBucket, srcObject, destBucket,
+	destObject string, srcInfo minio.ObjectInfo) (objInfo minio.ObjectInfo, err error) {
+	defer mon.Task()(&ctx)(&err)
+	o, err := s.storj.bs.GetObjectStore(ctx, srcBucket)
+	if err != nil {
+		return objInfo, err
+	}
+
+	rr, _, err := o.Get(ctx, paths.New(srcObject))
+	if err != nil {
+		return objInfo, err
+	}
+
+	defer func() {
+		if err := rr.Close(); err != nil {
+			// ignore for now
+		}
+	}()
+
+	r, err := rr.Range(ctx, 0, srcInfo.Size)
+	if err != nil {
+		return objInfo, err
+	}
+
+	defer func() {
+		if err = r.Close(); err != nil {
```

Comment:
There is bug on this line (missing `:`) that was addressed in #239. @cam-a could you review it and merge it, and then rebase, so we are not replicating this bug when copy-pasting the code.